packages/web/src/common/store/cache/collections/commonSagas.ts

```diff
@@ -780,39 +845,49 @@ function* fetchRepostInfo(entries) {
       })
       updates.push(followeeRepostUsers)
       if (subscriptionUids.length > 0) {
-        subscriptions.push({
-          id: entry.id,
-          kind: Kind.USERS,
-          uids: subscriptionUids
-        })
+        subscriptions.concat(
+          subscriptionUids.map((uid) => ({
+            id: entry.id,
+            kind: Kind.USERS,
+            uid
+          }))
+        )
```

Comment:
functional change

map the subscription uids into a list of subscription info, 1 per uid

Reply:
how did this even work before? lol

Reply:
bruh idk. Maybe it didn't? Hanging subscriptions is a pretty invisible bug